vpc/service/db/test/test_util.go

```diff
@@ -114,7 +117,26 @@ func StartPostgresContainer(ctx context.Context) (*PostgresContainer, error) {
 // Connect to the test DB
 func (c *PostgresContainer) Connect(ctx context.Context) (*sql.DB, error) {
 	url := fmt.Sprintf("postgres://pgtest:%s@localhost:%s/pgtest?sslmode=disable", c.password, c.port)
-	return sql.Open("postgres", url)
+
+	var err error
+	// Connect to DB and set up tables. Retry a couple of times in case of failure.
+	for i := 0; i < 10; i++ {
```

Comment:
Is it worth merging the version bump without this retry loop since the bug is supposedly fixed in the library?

Reply:
I don't think it was fixed completely. After bumping the version, the rate of failure decreased but I still saw it once in 5 runs. That issue in `lib/pq` is still open. Someone asked if it was fixed last year but no reply. I asked again. Before we have a definite answer, it's safer to just retry on our side.

Reply:
Ok, thanks.